tgui-next/packages/tgui/components/TitleBar.js

```diff
@@ -27,7 +28,7 @@ export const TitleBar = props => {
         color={statusToColor(status)}
         name="eye" />
       <div className="TitleBar__title">
-        {title}
+        {toTitleCase(title)}
```

Comment:
Have you checked if this plays nicely with things like "PanD.E.M.I.C"?

Reply:
I'll have to check. Probably would make the most sense to not titlecase it if it already contains caps.

Reply:
Or huge refactoring.

Reply:
Fixed now.